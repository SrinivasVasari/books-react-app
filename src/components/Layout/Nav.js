import React from "react";

const Nav = () => {
  return (
    <nav>
      <div className="nav-wrapper grey">
        <div className="container">
          <a href="!#" className="brand-logo">
            <i className="large material-icons">import_contacts</i>Books App
          </a>
          <ul id="nav-mobile" className="right hide-on-med-and-down">
            <li>
              <a href="!#">Books</a>
            </li>
            <li>
              <a href="!#">Add Book</a>
            </li>
            <li>
              <a href="!#">Edit Book</a>
            </li>
          </ul>
        </div>
      </div>
    </nav>
  );
};

export default Nav;
