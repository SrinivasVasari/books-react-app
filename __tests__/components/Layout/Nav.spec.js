import React from "react";
import * as renderer from 'react-test-renderer';
import Nav from "../../../src/components/Layout/Nav";

describe("Navigation", () => {
  it('should render without errors', () => {
    const component = renderer.create(<Nav />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
