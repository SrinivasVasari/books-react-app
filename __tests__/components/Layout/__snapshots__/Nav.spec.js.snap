// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation should render without errors 1`] = `
<nav>
  <div
    className="nav-wrapper grey"
  >
    <div
      className="container"
    >
      <a
        className="brand-logo"
        href="!#"
      >
        <i
          className="large material-icons"
        >
          import_contacts
        </i>
        Books App
      </a>
      <ul
        className="right hide-on-med-and-down"
        id="nav-mobile"
      >
        <li>
          <a
            href="collapsible.html"
          >
            Books
          </a>
        </li>
        <li>
          <a
            href="sass.html"
          >
            Add Book
          </a>
        </li>
        <li>
          <a
            href="badges.html"
          >
            Edit Book
          </a>
        </li>
      </ul>
    </div>
  </div>
</nav>
`;
